.github/workflows/ci_dockerfile_runner.yaml

```diff
@@ -0,0 +1,40 @@
+name: CI GitHub Runners
```

Comment:
Change this name to separate form the other workflows. Something like `ci-Ubuntu22`